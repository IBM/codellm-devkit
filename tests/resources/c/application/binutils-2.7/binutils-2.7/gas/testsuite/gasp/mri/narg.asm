loop	macro	arg1,arg2,arg3
	dc.l	NARG
	ifne	NARG
	dc.l	arg1
	loop	arg2,arg3
	endc
	endm

	loop	1,2,3
