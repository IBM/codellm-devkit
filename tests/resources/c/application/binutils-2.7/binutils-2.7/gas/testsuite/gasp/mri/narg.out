;loop	macro	arg1,arg2,arg3
;	dc.l	NARG
;	ifne	NARG
;	dc.l	arg1
;	loop	arg2,arg3
;	endc
;	endm
;

;	loop	1,2,3
;	dc.l	3
	dc.l	3
;	ifne	3
;	dc.l	1
	dc.l	1
;	loop	2,3
;	dc.l	2
	dc.l	2
;	ifne	2
;	dc.l	2
	dc.l	2
;	loop	3,
;	dc.l	1
	dc.l	1
;	ifne	1
;	dc.l	3
	dc.l	3
;	loop	,
;	dc.l	0
	dc.l	0
;	ifne	0
;	dc.l	
;	loop	,
;	endc
;	endc
;	endc
;	endc
;